pkg/web/packrd/packed-packr.go

```diff
@@ -0,0 +1,48 @@
+// +build !skippackr
```

Comment:
Was this expected to be part of the commit?

Reply:
blunder! good catch. Rebasing now